stable/prometheus/values.yaml

```diff
@@ -328,6 +328,11 @@ server:
     #     hosts:
     #       - prometheus.domain.com
 
+  ## Node tolerations for server scheduling to nodes with taints
+  ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
+  ##
+  tolerations: []
```

Comment:
Could you add suitable example configuration like below?
```
tolerations: []
  #- key: "key"
  #operator: "Equal|Exists"
  #value: "value"
  #effect: "NoSchedule|PreferNoSchedule|NoExecute(1.6 only)"
```

Please take a look at this `./stable/traefik/values.yaml` for reference